### PR TITLE
multi cluster landing page

### LINKED ISSUE #543: Document a multi-cluster setup
### Description of problem

We currently have some good examples of a [supply chain](https://github.com/vmware-tanzu/cartographer/blob/main/examples/gitwriter-sc/README.md) and a [delivery](https://github.com/vmware-tanzu/cartographer/blob/main/examples/basic-delivery/README.md
) that show how you would support gitops across multiple clusters. We also have a [really good diagram](https://cartographer.sh/docs/v0.1.0/architecture/#complete-supply-chain-and-delivery-with-gitops
) that shows what a multi-cluster setup looks like. It would be nice if we had a landing page in our docs that aggregates all this information so when people ask about multi-cluster support we can point them to a single place.

### PR DESCRIPTION
**Pull request by emmjohnson**
Co-authored-by: Marty Spiewak <mspiewak@vmware.com>

[PR requirements]: https://github.com/vmware-tanzu/cartographer/blob/main/CONTRIBUTING.md#commit-message-and-pr-guidelines

## Changes proposed by this PR

closes #543 

## Release Note

[imperative mood]: https://chris.beams.io/posts/git-commit/#imperative

## PR Checklist

Note: Please do not remove items. Mark items as done `[x]` or use ~strikethrough~ if you believe they are not relevant

- [ ] Linked to a relevant issue. Eg: `Fixes #123` or `Updates #123`
- [ ] Removed non-atomic or `wip` commits
- [ ] Filled in the [Release Note](#Release-Note) section above 
- [ ] Modified the docs to match changes

### CODE REVIEW

**Comment by jwntrs** [comment]
It would be nice to add some notes about how we expect promotion to work between environments as per @zrob note here: https://github.com/vmware-tanzu/cartographer/issues/543#issuecomment-1045049832

We want to describe something along the lines of:
1. supply chain builds config and commits to a branch `staging/feature-a`
2. a human PRs branch `staging/feature-a` into `staging`
3. those changes get deployed to staging env
4. a task at the end of the staging delivery commits to something like `prod/feature-a` (essentially promoting those changes)
5. a human PRs changes from `prod/feature-a` into `prod`
6. those changes get deployed on prod

File: site/content/docs/development/multi-cluster.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Cartographer helps pave a path to production across clusters with [ClusterSupplyChains](reference/workload.md#clustersupplychain)
+and [ClusterDeliveries](reference/deliverable.md#clusterdelivery), leveraging [GitOps](https://www.gitops.tech/).
+
+A ClusterSupplyChain can be installed in the development cluster to manage the path from source code to Kubernetes
+configuration in a Git repository. Meanwhile, a ClusterDelivery can be installed in the production cluster, which
+will pick up that configuration from the Git repository to be deployed and tested.
+
+![Multi-Cluster](../img/multi-cluster.jpg)
+
+### Example PR based flow
```
Comment (emmjohnson) [inline site/content/docs/development/multi-cluster.md:14]: @rawlingsj any strong opinions on this workflow before we draw up some diagrams?

File: site/content/docs/development/multi-cluster.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+0. Developer commits new source code
+1. ClusterSupplyChain in the development cluster generates Kubernetes configuration and commits it to a
+   git repository/branch (i.e. staging/feature-a)
+2. "Staging environment maintainer" opens a PR and merges staging/feature-a into staging
+3. ClusterDelivery in the staging cluster picks up the merged PR and deploys the Kubernetes configuration to be tested and commits
+   it to a git repository/branch (i.e. production/feature-a)
```
Comment (rawlingsj) [inline site/content/docs/development/multi-cluster.md]: I've always preferred to have a separate git repository for environments / clusters and "promote"  into these.  The main reason for that is folks tend to have different owners or approvers for different clusters.  Keeping them separate means we can control RBAC via the repo for who is allowed to merge a PR to staging, and a different set of users that own production merges.  It also helps avoid accidentally merging to the wrong branch which could mean production getting changes that were meant for staging.

File: site/content/docs/development/architecture.md [hunk 3]
```diff
@@ -140,10 +140,6 @@ When Cartographer reconciles an owner, each resource in the matching blueprint i
 
 ![Realize](../img/realize.jpg)
 
-### Complete Supply Chain and Delivery with GitOps
```
Comment (jwntrs) [inline site/content/docs/development/architecture.md:143]: Maybe we should keep this here as well?

**Comment by zrob** [comment]
yay! glad to see us starting to get some guidance here!

- consider changing the "development" cluster name to "build" since that's more what's happening

- would be nice to see some description on the why and role for the different clusters. ie - you want an isolated build cluster. then want to promote the same artifact through multiple environments to first test/validate, then run in production.

-

File: site/content/docs/development/architecture.md (hunk 3 above)
Comment (emmjohnson) [reply site/content/docs/development/architecture.md:143]: We thought it looked out of place....

File: site/content/docs/development/multi-cluster.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# Multi-Cluster
+
+## Overview
+It is often desirable to separate build and production environments; this can be achieved with multiple clusters.
+Cartographer helps pave a path to production across clusters with [ClusterSupplyChains](reference/workload.md#clustersupplychain)
+and [ClusterDeliveries](reference/deliverable.md#clusterdelivery), leveraging [GitOps](https://www.gitops.tech/).
+
+A ClusterSupplyChain can be installed into an isolated build cluster to manage the path from source code to Kubernetes
+configuration in a Git repository. Meanwhile, a ClusterDelivery can be installed in the production cluster, which
+will pick up that configuration from the Git repository to be deployed and tested. This allows for the same artifact to
```
Comment (cirocosta) [inline site/content/docs/development/multi-cluster.md:10]: I wasn't super sure about the 

> to be deployed and *tested*

but no strong opinions really

**Review by cirocosta — APPROVED** [review]
lgtm! read it via the deploy preview 👍 👍 I'd wait for @zrob and @rawlingsj review before merging though

**Comment by emmjohnson** [comment]
> lgtm! read it via the deploy preview 👍 👍 I'd wait for @zrob and @rawlingsj review before merging though

Going to merge. We can always make changes 😃

**Pull request merged**